apps/agile-compose

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+export AGILE_CLIENT_PORT=2000
```

Comment:
I'd break the declaration of all envars into a separate file. This file should also get executed at build time (then if we add environment variable via the resin dashboard we can overwrite them) this makes cycles faster if you just want to change some config.